Description: Changes meant for version 1.5.0 but missing.
 Removed references to obsoleted 'soundtouch-1.4.pc.in' replacing these with
 'soundtouch.pc.in'.
 .
 Bump version from 1.4.0 to 1.5.0.
 .
 Remove when version > 1.5.0 is released.
Forwarded: not-needed
Author: Olli Parviainen <oparviai@iki.fi> 
Applied-Upstream: http://soundtouch.svn.sourceforge.net/viewvc/soundtouch?view=revision&revision=89
Last-Update: 2011-02-25
--- soundtouch-1.5.0.orig/Makefile.am
+++ soundtouch-1.5.0/Makefile.am
@@ -1,6 +1,6 @@
 ## Process this file with automake to create Makefile.in
 ##
-## $Id: Makefile.am 38 2008-12-25 17:00:23Z oparviai $
+## $Id$
 ##
 ## Copyright (C) 2003 - David W. Durham
 ## 
@@ -52,7 +52,7 @@ EXTRA_DIST=                  \
        README.html
 
 pkgconfigdir = $(libdir)/pkgconfig
-pkgconfig_DATA = soundtouch-1.4.pc
+pkgconfig_DATA = soundtouch.pc
 
 ## This removes stuff from the distribution which may be present
 ## from a cvs checkout or other build reasons
--- soundtouch-1.5.0.orig/configure.ac
+++ soundtouch-1.5.0/configure.ac
@@ -1,6 +1,6 @@
 dnl SoundTouch configure.ac, by David W. Durham
 dnl
-dnl $Id: configure.ac 38 2008-12-25 17:00:23Z oparviai $
+dnl $Id$
 dnl
 dnl This file is part of SoundTouch, an audio processing library for pitch/time adjustments
 dnl 
@@ -19,7 +19,7 @@ dnl this program; if not, write to the F
 dnl Place - Suite 330, Boston, MA  02111-1307, USA
 # Process this file with autoconf to produce a configure script.
 
-AC_INIT(SoundTouch, 1.4.0, [http://www.surina.net/soundtouch])
+AC_INIT(SoundTouch, 1.5.0, [http://www.surina.net/soundtouch])
 AC_CONFIG_AUX_DIR(config)
 AM_CONFIG_HEADER([include/soundtouch_config.h])
 AM_INIT_AUTOMAKE
@@ -129,7 +129,7 @@ AC_CONFIG_FILES([
 ])
 
 AC_OUTPUT(
-	soundtouch-1.4.pc
+	soundtouch.pc
 )
 
 dnl use 'echo' to put stuff here if you want a message to the builder
